src/java.xml/share/classes/javax/xml/xpath/XPathFactory.java

```diff
@@ -369,4 +370,57 @@ public abstract boolean getFeature(String name)
     * @return New instance of an <code>XPath</code>.
     */
     public abstract XPath newXPath();
+
+    /**
+     * Sets a property for this {@code XPathFactory}. The property applies to
+     * {@code XPath} objects that the {@code XPathFactory} creates. It has no
+     * impact on {@code XPath} objects that are already created.
+     *
+     * @implSpec
+     * The default implementation throws
+     * {@link java.lang.UnsupportedOperationException}.
+     *
+     * @param name the property name
+     * @param value the value for the property
+     *
+     * @throws IllegalArgumentException if the property name is not recognized,
+     * or the value can not be assigned
+     * @throws UnsupportedOperationException if the implementation does not
+     * support the method
+     * @throws NullPointerException if the {@code name} is {@code null}.
```

Comment:
Drop the "." at the end of the sentence (to be consistent with the other throws).